File: demo01.py
#!/bin/usr/python

#Print a checker board of specified width and height
#Written by Raphael Rueda

import sys

print "enter width"
width = int(sys.stdin.readline())
print "enter height"
height = int(sys.stdin.readline())
pattern = 0

for i in range(0, height):
    if i % 2 == 0: 
	for j in range(0, width):
	    if j % 2 == pattern:
		sys.stdout.write("#")
	    else:
		sys.stdout.write(" ")
	if pattern == 0: 
	    pattern = 1
	else: 
	    pattern = 0
        print

File: demo01.pl
#!/usr/bin/perl -w

print "enter width", "\n";
$width = <STDIN>;
print "enter height", "\n";
$height = <STDIN>;
$pattern = 0;

foreach $i (0..$height - 1) {
    if ($i % 2 == 0) {
        foreach $j (0..$width - 1) {
            if ($j % 2 == $pattern) {
                print "#";
            } else {
                print " ";
            }
        }
        if ($pattern == 0) {
            $pattern = 1;
        } else {
            $pattern = 0;
        }
        print "\n";
    }
}
